ansible/roles/ansible-datamover-api/tasks/install_rhel.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Install packages on datamover api container | centos-release-openstack-ussuri
+  yum:
+    update_cache: yes
+    name:
+      - "centos-release-openstack-ussuri"
```

Comment:
can we make it independent of the releases ? so that we don't need to update the code everytime.
Instead we can maintain inventory variables per release.